master/buildbot/test/unit/test_contrib_buildbot_cvs_mail.py

```diff
@@ -118,9 +126,11 @@ class TestBuildbotCvsMail(unittest.TestCase):
         skip = ("'%s' does not exist (normal unless run from git)"
                 % buildbot_cvs_mail_path)
 
-    def assertOutputOk(self, xxx_todo_changeme4, regexList):
+    def assertOutputOk(self, result, regexList):
```

Comment:
I'm not sure what do we need `result` for: why not just replace it with `(output, code)`?